OKTA-413125 Overall style edit for OIE embedded auth content - part 1 (#2571)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/go/integrationsteps.md

<div class="common-image-format common-image-format-vertical-margin">

![Displays a basic sign-in page in a Golang application](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-basic-sign-on-page.png)

</div>

> During page load, call the `Client` object's `InitLogin()` method. This method returns a `LoginResponse` object that is used to initiate the sign-in process with Okta. This object also contains a list of available social Identity Providers (IdPs) that are discussed in more detail in the [Sign in with Facebook](/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/go/main/)

Reviewer: ```suggestion
During page load, call the `Client` object's `InitLogin()` method. This method returns a `LoginResponse` object that is used to initiate the sign-in process with Okta. This object also contains a list of available social Identity Providers (IdPs) that are discussed in more detail in [Sign in with Facebook](/docs/guides/oie-embedded-sdk-use-case-sign-in-soc-idp/go/main/)
```

}
```

> Next, using the `LoginResponse` object obtained from [step 2](#_2-capture-credentials-with-the-sign-in-page), call the `Identify()` method, passing in this new `IdentifyRequest` object.

Reviewer: ```suggestion
Next, you can use the `LoginResponse` object obtained from [step 2](#_2-capture-credentials-with-the-sign-in-page) to call the `Identify()` method, which passes in this new `IdentifyRequest` object.
```

### 4: Save the tokens and redirect the user to the default home page

> The `Identify()` method returns two objects: `LoginResponse` and `error`. Use the `error` object to determine if there were errors in the

Reviewer: ```suggestion
The `Identify()` method returns two objects: `LoginResponse` and `error`. Use the `error` object to determine if there are errors in the
```
